networks/p2p/server.go

```diff
@@ -577,8 +577,9 @@ func (srv *MultiChannelServer) setupConn(c *conn, flags connFlag, dialDest *disc
 			dialDest.TCPs = append(dialDest.TCPs, uint16(listenPort))
 		}
 		srv.AddPeer(dialDest)
```

Comment:
```suggestion
```

Reply:
Thanks for the comment. I deleted it. PTAL :)